cmd/controller/app/controller.go

```diff
@@ -29,7 +29,7 @@ import (
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
-	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	clientv1 "k8s.io/client-go/kubernetes/typed/core/v1"
```

Comment:
My hacky bash script (see next commit) replaces v1alpha2 imports with `v1`, but that sometimes clashes with the corev1 imports. So I'm renaming those.